sale_purchase_order_line_report/models/product_product.py

```diff
@@ -10,7 +10,7 @@ class ProductProduct(models.Model):
 
     def _get_action_view_domain(self):
         return [('product_id', '=', self.id)]
-    
+
```

Comment:
Remove this change. It seems to be something right, but it shouldn't belong to this commit